Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -152,6 +151,47 @@ private static void ShowPlayerResult(Dictionary<string, int> playerResult)
                 WriteColoredMessage($" { playerResults.Key} -> {playerResults.Value}", color);
             }
         }
+        private static void StartMainGame()
+        {
+            string repeatGame = "y";
+
+            while(repeatGame == "y")
+            {
+                ShowGameNumber();
+
+                Dictionary<string, int> playerResult = StartGame();
+
+                ShowPlayerResult(playerResult);
+
+                SeperateGames();
+
+                AskRepeat();
+
+                string repeatAgainGame = Console.ReadLine();
```

Comment:
maybe lines 
```
string repeatAgainGame = Console.ReadLine();
repeatGame = repeatAgainGame;
```
should go into AskRepeast method ? as it is related with repeat logic
and AskRepeat method can return chosenRepeatOption variable (string)

Reply:
And better name i think for this method would be: `ConfirmRepeat` what do you think ? (confirm is more programatic word than ask)

Reply:
well i cant because repeatgame variable is stored in StartMainGame method so i can not store this in another method unless i put it on field then i could use it in two methods  what do you think :)